project/CrossArchitecture.scala

```diff
@@ -0,0 +1,69 @@
+import sbt._
+import Keys.{name, unmanagedResourceDirectories, baseDirectory}
+import sbtcrossproject._
+import scala.scalanative.build
+import build.{Bits, ThirtyTwo, SixtyFour, TargetArchitecture}
+import scala.scalanative.sbtplugin.ScalaNativeCrossVersion
+import scala.scalanative.sbtplugin.ScalaNativePlugin
+import scala.scalanative.sbtplugin.ScalaNativePlugin.autoImport._
+
+case class CrossBitsPlatform(bits: Bits, lib: Boolean = false)
```

Comment:
Given that we can only ever have two instances, lets make this sealed with two objects inheriting from it.